src/esp/gfx/Simulator.cpp

```diff
@@ -373,6 +373,8 @@ double Simulator::getWorldTime() {
 
 bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
                                  const nav::NavMeshSettings& navMeshSettings) {
+  CHECK(config_.createRenderer);
```

Comment:
```suggestion
  CORRADE_ASSERT(config_.createRenderer,
                 "Meshes were not loaded, cannot recompute navmesh.  Please "
                 "add a sensor to the agent(s).",
                 {});
```

Let's add an error message!

Reply:
Yes, we prefer to use CORRADE_ASSERT.
Also, add `Simulator::recomputeNavMesh: `  to the beginning of the message body.

Thanks!